qemu/tests/block_hotplug.py

```diff
@@ -87,6 +87,7 @@ def run_sub_test(params, plug_tag):
         plug_tag = "before_plug"
         run_sub_test(params, plug_tag)
 
+        session = vm.wait_for_login(timeout=timeout)
```

Comment:
When hotplug one-pci for 200 times, will also have this issue.

Reply:
Thanks for your comments, I will correct it.

Reply:
I misunderstand, however many times, the ssh session will be created then closed sequentially, not parallel.